packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -251,8 +253,25 @@ describeNoCompat("Container", (getTestObjectProvider) => {
         });
 
         container.forceReadonly(true);
-        assert.strictEqual(container.readonly, true);
+        assert.strictEqual(container.readOnlyInfo.readonly, true);
 
         assert.strictEqual(runCount, 1);
     });
+
+    it("closeAndGetPendingLocalState() called on container", async () => {
```

Comment:
After removing the call to mess with the delta manager, this test isn't as relevant as it was in the previous iteration. However, it would add some fractional value in terms of code coverage. Let me know if you have strong feelings for removing this test altogether from this PR, otherwise I'd leave it in.

Reply:
I think it's fine as is